### ISSUE TITLE
Vim keyboard shortcut help pane not screen-reader accessible

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6123]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.798
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- with Vim mode enabled in editor, type `:help`
- attempt to use the resulting popup pane via screen-reader

### Describe the problem in detail
- any attempt to use the pane results in it closing

### Describe the behavior you expected
- should be able to get Vim help, either directly in this panel, or by having a separate web-page that is screen-reader accessible, similar to what is done for main keyboard-shortcut help
